src/v2/Apps/_AuctionPartnerships2/AuctionPartnershipsApp.tsx

```diff
@@ -1,9 +1,378 @@
-import { Text } from "@artsy/palette"
+import {
+  Text,
+  Box,
+  Image,
+  GridColumns,
+  Column,
+  Join,
+  Spacer,
+  Flex,
+  Avatar,
+} from "@artsy/palette"
+import { Fragment } from "react"
+import { MetaTags } from "v2/Components/MetaTags"
 
 export const AuctionPartnershipsApp: React.FC = () => {
   return (
     <>
-      <Text variant="xl">Auction Partnerships App</Text>
+      <MetaTags
+        title="Auction Partnerships"
+        description=""
+        pathname="/auction-partnerships"
+      />
+
+      <Join
+        separator={
+          <Spacer mt={6} mb={6} border="1px solid" borderColor="black15" />
```

Comment:
You could use a `<Separator my={6} />` here

Reply:
right! `my` works for both the directions on the y axis